.travis.yml

```diff
@@ -16,7 +16,7 @@ addons:
       - python3-yaml
 install: ./.travis/setup_lobby_database
 before_script: ./.travis/setup_gpg
-script: ./gradlew check jacocoJunit5TestReport
+script: ./gradlew check jacocoJunit5TestReport --parallel
```

Comment:
Nit: For consistency with other Gradle commands in this file, canonical Gradle usage is to place options before tasks:

```
USAGE: gradlew [option...] [task...]
```